app/views/docs/realtime.phtml

```diff
@@ -0,0 +1,341 @@
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client()
+val realtime = Realtime(client)
+
+val subscription = realtime.subscribe("files", callback = { param ->
+    // Callback will be executed on changes for all files.
+    print(param.toString())
+})
+
+// Closes the subscription.
+subscription()</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="/docs/realtime#payload" id="payload">Payload</a></h2>
+
+<p>The payload from the subscription will contain following properties:</p>
+
+<table class="vertical full args">
+    <thead>
+        <tr>
+            <td style="width: 140px">Name</td>
+            <td style="width: 120px">Type</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td data-title="Name: ">
+                event
+            </td>
+            <td data-title="Type: ">
+                string
+            </td>
+            <td data-title="Description: ">
+                The <a href="/docs/webhooks#events">system event</a> that triggered this update.
+            </td>
+        </tr>
+        <tr>
+            <td data-title="Name: ">
+                channels
+            </td>
+            <td data-title="Type: ">
+                string[]
+            </td>
+            <td data-title="Description: ">
+                An array of <a href="/docs/realtime#channels">channels</a> that can receive this message.
+            </td>
+        </tr>
+        <tr>
+            <td data-title="Name: ">
+                timestamp
+            </td>
+            <td data-title="Type: ">
+                integer
+            </td>
+            <td data-title="Description: ">
+                The UNIX timestamp from the server to ensure consistency across all client platforms and real-time technologies.
+            </td>
+        </tr>
+        <tr>
+            <td data-title="Name: ">
+                payload
+            </td>
+            <td data-title="Type: ">
+                object
+            </td>
+            <td data-title="Description: ">
+                Payload contains the data equal to the response model.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<p>If you subscribe to the <span class="tag">documents</span> channel and a document the user is allows to read is updated, you will receive an object containing informations about the event and also the updated document.</p>
+
+<p>The response will look like this:</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+    <pre class="line-numbers"><code class="prism language-json" data-prism>{
+    "event": "database.documents.update",
+    "channels": [
+        "documents",
+        "documents.[DOCUMENT_ID]",
+        "collections.[COLLECTION_ID].documents"
+    ],
+    "timestamp": 1629719169,
+    "payload": {
+        "$id": "[DOCUMENT_ID]",
+        "$collection": "[COLLECTION_ID]",
+        "$permissions": [],
```

Comment:
Note: we need to update this example after the DB refactor. `$permissions` will be replaced with `$read` and `$write`